include/zephyr/drivers/usb_c/usbc_pd.h

```diff
@@ -53,6 +53,55 @@ extern "C" {
  */
 #define PD_MAX_EXTENDED_MSG_CHUNK_LEN    26
 
+/**
+ * @brief The CapsCounter is used to count the number of Source_Capabilities
+ *	  Messages which have been sent by a Source at power up or after a
+ *	  Hard Reset.
+ *	  See Table 6-63 Counter parameters
+ *	  Parameter Name: CapsCounter
+ */
+#define PD_N_CAPS_COUNT 50
+
+/**
+ * @brief The HardResetCounter is used to retry the Hard Reset whenever there
+ *	  is no response from the remote device (see Section 6.6.6)
+ *	  See Table 6-63 Counter parameters
+ *	  Parameter Name: HardResetCounter
+ */
+#define PD_N_HARD_RESET_COUNT 2
+
+/**
+ * @brief The NoResponseTimer is used by the Policy Engine in a Source to
+ *	  determine that its Port Partner is not responding after a Hard Reset.
+ *	  See Table 6-61 Time Values
+ *	  Parameter Name: NoResponseTimer
+ */
+#define PD_T_NO_RESPONSE_MIN_MS 4500
+
+/**
+ * @brief The NoResponseTimer is used by the Policy Engine in a Source to
+ *	  determine that its Port Partner is not responding after a Hard Reset.
+ *	  See Table 6-61 Time Values
+ *	  Parameter Name: NoResponseTimer
+ */
+#define PD_T_NO_RESPONSE_MAX_MS 5500
+
+/**
+ * @brief Minimum time a source shall wait before sending a
+ *	  Source_Capabilities message
```

Comment:
Probably add that this is for the timeframe before an explicit contract

Reply:
done